src/components/Questions/Body.vue

```diff
@@ -354,6 +388,15 @@ export default defineComponent({
       { deep: true }
     );
 
+    watch(
+      () => props.draftAnswer,
+      (newValue) => {
+        if (typeof newValue == "string" || newValue == null) {
```

Comment:
instead of checking if the answer's type is string, shouldn't we check if the current question is a subjective one?

this is the intention here right? to see if the changed draft answer belongs to subjective question or not?

Reply:
comment saying that it is only for subjective